SPECS/CUnit/CUnit.spec

```diff
@@ -0,0 +1,197 @@
+%global tarver 2.1-3
+
+Name:           CUnit
+Version:        2.1.3
+Release:        23%{?dist}
+Summary:        Unit testing framework for C
+Vendor:         Microsoft
```

Comment:
Please use "Microsoft Corporation" instead of "Microsoft".

Reply:
Done